packages/core/src/dependency-finder/exec-command.ts

```diff
@@ -0,0 +1,14 @@
+import { execSync } from "child_process";
+
+export function execCommand(command: string, cwd: string): string {
+  try {
+    const output = execSync(command, {
+      cwd,
+      encoding: "utf-8",
+    });
+
+    return output;
+  } catch (error) {
+    throw new Error("Error executing command");
+  }
```

Comment:
```suggestion
  } catch (error) {
    if (error instanceof Error) {
      throw new Error(
        `Error executing command ${command}: ${error.name} - ${error.message}`,
      );
    }

    throw new Error(`Error executing command ${command}`);
  }
```

The more descriptive the error the better for the end user